### ISSUE TITLE
Accessibility Issue - Scrollpanel scrollbar is focusable but remains hidden and has no visible focus indicator

### ISSUE THREAD

**Issue by emma-agility** [primefaces/primeng#19621]
As per WCAG SC 2.4.7 any element that is focusable by keyboard needs to have a visible focus indicator. Since the scrollbar remains hidden when focused and has no visible focus indicator that is a violation of the SC. Please either allow us to prevent the scrollbar from being focusable (its not really necessary anyways since you can already scroll with arrow keys or by tabbing through the page without focusing the scrollbar) or make the scrollbar visible when focused and add a focus indicator.
